src/components/Hero/Hero.stories.mdx

```diff
@@ -0,0 +1,274 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import Hero from "./Hero";
+import { HeroTypes } from "./HeroTypes";
+import Image from "../Image/Image";
+import Heading from "../Heading/Heading";
+import sections from "../../utils/siteSections";
+import { action } from "@storybook/addon-actions";
+
+<Meta
+  title="Hero"
+  component={Hero}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10968%3A5314",
+    },
+  }}
+  argTypes={{
+    attributes: { table: { disable: true } },
+    blockName: { table: { disable: true } },
+    className: { table: { disable: true } },
+    heading: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+    subHeaderText: { table: { disable: true } },
+    image: { table: { disable: true } },
+  }}
+/>
+
+# Hero
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.2.0`    |
+| Latest            | `0.23.4`   |
+
+<Description of={Hero} />
+
+The `Hero` component is used to display a full width banner at the top of a page. The `Hero` will contain a required `h1` page title and may also include optional descriptive text and images.
+
+<Preview withToolbar>
+  <Story
+    name="Hero Props"
+    args={{
+      heroType: HeroTypes.Primary,
+    }}
+  >
+    {args =>
+      (args.heroType === HeroTypes.Primary && (
+        <Hero
+          {...args}
+          heading={
+            <Heading
+              level={1}
+              id={"1"}
```

Comment:
Nits: You can remove the curly braces when the prop is a string literal.

Reply:
Done.